config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/designate-operator
-  newTag: latest
+  newName: registry-proxy.engineering.redhat.com/rh-osbs/rhosp-dev-preview-trunk-designate-rhel9-operator
```

Comment:
@yazug , is this needed?

Reply:
no not needed, reverted that change and rebased to current. Thanks for spotting that.